Maddori.Apple/Maddori.Apple/Screen/AddFeedbackContent/AddFeedbackContentViewController.swift

```diff
@@ -153,17 +196,67 @@ final class AddFeedbackContentViewController: BaseViewController {
             $0.top.equalTo(feedbackContentLabel.snp.bottom).offset(SizeLiteral.labelComponentPadding)
             $0.leading.equalTo(addFeedbackContentView.snp.leading).inset(SizeLiteral.leadingTrailingPadding)
             $0.trailing.equalTo(addFeedbackContentView.snp.trailing).inset(SizeLiteral.leadingTrailingPadding)
-            $0.height.greaterThanOrEqualTo(150)
+            $0.height.equalTo(150)
+        }
+        
+        addFeedbackContentView.addSubview(feedbackStartSwitch)
+        feedbackStartSwitch.snp.makeConstraints {
+            $0.top.equalTo(feedbackContentTextView.snp.bottom).offset(SizeLiteral.componentIntervalPadding)
+            $0.trailing.equalTo(addFeedbackContentView.snp.trailing).inset(SizeLiteral.leadingTrailingPadding)
```

Comment:
p3;

```suggestion
            $0.trailing.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
```
addFeedbackContentView이게 사실 superView라 `equalToSuperview`로 해줘도 됩니다 !

Reply:
[e4aab51](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/47/commits/e4aab5173c8b639771a8c4092987ea5ddf08fff0)
수정해서 반영했습니다 ! 감사합니다 :)